apps/dsb-client-gateway-scheduler/src/app/modules/clients/service/clients.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable, Logger, OnApplicationBootstrap } from '@nestjs/common';
+import { ClientsService as ClientsLibService } from '@dsb-client-gateway/ddhub-client-gateway-clients';
+import { CronJob } from 'cron';
+import {
+  ClientEntity,
+  CronJobType,
+  CronStatus,
+  CronWrapperRepository,
+} from '@dsb-client-gateway/dsb-client-gateway-storage';
+import { SchedulerRegistry } from '@nestjs/schedule';
+import { ConfigService } from '@nestjs/config';
+import moment from 'moment';
+
+@Injectable()
+export class ClientsService implements OnApplicationBootstrap {
+  protected readonly logger = new Logger(ClientsService.name);
+
+  constructor(
+    protected readonly clientsLibService: ClientsLibService,
+    protected readonly schedulerRegistry: SchedulerRegistry,
+    protected readonly cronWrapper: CronWrapperRepository,
+    protected readonly configService: ConfigService
+  ) {}
+
+  public async onApplicationBootstrap(): Promise<void> {
+    const isCronEnabled: boolean = this.configService.get<boolean>(
+      'CLIENTS_CRON_ENABLED',
+      true
+    );
+
+    if (!isCronEnabled) {
+      this.logger.warn(`Channel did cron job is disabled`);
```

Comment:
Is this the right warning text, should it be Client cron is disabled?

Reply:
thank you, changed already